src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -336,7 +334,7 @@ private class AppTaskLauncherActor(
         case Some(CreatedTask(mesosTask, marathonTask)) =>
           def updateActorState(): Unit = {
             tasksMap += marathonTask.getId -> marathonTask
-            inFlightTaskLaunches += mesosTask.getTaskId -> None
+            inFlightTaskLaunches += mesosTask.getTaskId.getValue -> None
```

Comment:
Not sure if I'm nit-picking or if this is on purpose: `marathonTask.getId` is the same as `mesosTask.getTaskId.getValue` – to make it obvious that both keys are the same, I'd rather write
```
tasksMap += marathonTask.getId -> marathonTask
inFlightTaskLaunches += marathonTask.getId -> None
```